modules/ixBidAdapter.js

```diff
@@ -210,16 +243,30 @@ function buildRequest(validBidRequests, bidderRequest, impressions, version) {
   // Always use secure HTTPS protocol.
   let baseUrl = SECURE_BID_URL;
 
+  // Dict for identity partners already populated from prebid
```

Comment:
This is a great development! I am curious however, why only liveramp? prebid might also populate liveintent, tdid and others your endpoint already understands. Maybe i am confused but this comment says partners and the code seems to only be for liveramp.

Reply:
We plan to add in additional partners, so we're just starting with LiveRamp.  We want to make sure we build out toggles on the server side, so DSPs can control which IDs get passed down to them (which is different than matching an ID against an audience) regardless of what gets consumed on the page by the IX adapter. 

We already have these ID DSP toggles for LR, so it was an easy starting point.  Over the next month or two, you'll see us consume the other IDs.

Reply:
looking forward to that, ty!

Reply:
this is great to see merged in! just to confirm @sulleh ; this seems to allow one to fully support roundel from within prebid?